projects/elonkit/src/ui/kbd/kbd.component.theme.scss

```
@@ -0,0 +1,17 @@
+
+@mixin es-kbd-theme($theme, $elonkit-theme) {
+  $mono-a: map-get($elonkit-theme, mono-a);
+  .key {
+    box-shadow: mat.get-color-from-palette($mono-a, A200) 0 0 0 1px inset;
+
+    &.raised {
```

Вообще раньше в ангуляре придердживались БЭМ, модификаторы отделяли через `_`
```suggestion
    &_raised {
```